pyanaconda/modules/storage/partitioning/manual/manual_partitioning.py

```diff
@@ -150,22 +150,39 @@ def _recreate_btrfs_volume(self, storage, device):
         storage.create_device(new_btrfs)
         return new_btrfs
 
+    def _recreate_btrfs_subvolume(self, storage, device):
+        """Recreate a btrfs subvolume device by destroying and adding it.
+
+        :param storage: an instance of the Blivet's storage object
+        :param dev_spec: a BtrfsSubVolumeDevice to recreate
```

Comment:
the param is `device`, please update it also for `_recreate_btrfs_volume` when we are on it.